### PR TITLE
Adds warning stating that MariaDB 10.2 will not be guaranteed support in future patches

### PR DESCRIPTION
**Pull request by GuptaManan100**
## Description

Documentation for https://github.com/vitessio/vitess/pull/11074 and https://github.com/vitessio/vitess/pull/11073

### CODE REVIEW

File: content/en/docs/12.0/overview/supported-databases.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Vitess supports the core features of MariaDB versions 10.0 to 10.3. Vitess [does
 
 {{< warning >}}MariaDB support has been EOL'd in Vitess 14.0.{{< /warning >}}
 
+{{< warning >}}MariaDB 10.2 EOL'd on 23 May, 2022 and Vitess patch releases after 12.0.5 will not guarantee support for it. {{< /warning >}}
```
Comment (mattlord) [inline content/en/docs/12.0/overview/supported-databases.md]:
```
{{< warning >}}[MariaDB 10.2 was EOL'd on May 23, 2022](https://mariadb.org/about/#maintenance-policy) and thus Vitess support for it is also EOL in 12.0.5 and later.{{< /warning >}}
```

**Review by mattlord — COMMENTED** [review]
I think we can simplify the note and we should add a link. Otherwise it's spot on.

File: content/en/docs/12.0/overview/supported-databases.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/docs/12.0/overview/supported-databases.md]: This is much better 🙇‍♂️

Comment (GuptaManan100) [reply content/en/docs/12.0/overview/supported-databases.md]: Fixed

**Review by mattlord — APPROVED** [review]
Thanks!

**Review by rsajwani — APPROVED** [review]

**Pull request merged**